src/Controls/tests/TestCases.HostApp/Issues/Issue23411.xaml.cs

```diff
@@ -0,0 +1,43 @@
+﻿#if IOS
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Controls.PlatformConfiguration;
+using Microsoft.Maui.Controls.PlatformConfiguration.iOSSpecific;
+using Application = Microsoft.Maui.Controls.Application;
+using NavigationPage = Microsoft.Maui.Controls.NavigationPage;
+
+namespace Maui.Controls.Sample.Issues
+{
+	[Issue(IssueTracker.Github, 23411, "[iOS]Use non-overridden traits in AppInfoImplementation.RequestTheme",
+		PlatformAffected.iOS)]
+	public partial class Issue23411 : ContentPage
+	{
+		public Issue23411()
+		{
+			InitializeComponent();
+			Application.Current.UserAppTheme = AppTheme.Dark;
+		}
+
+		private void OnModalPage(object sender, EventArgs e)
```

Comment:
The EventHandler is only created for iOS and failing in the rest of the platforms.

```
D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue23411.xaml(30,17): XamlC error XC0002: EventHandler "OnModalPage" with correct signature not found in type "Maui.Controls.Sample.Issues.Issue23411". [D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-windows10.0.20348.0]
    5 Error(s)

```

Reply:
@jsuarezruiz ,Thank you for sharing your concerns with us. We are actively working to resolve the issue and optimize the code for all platforms. We will provide you with an update on the latest fix as soon as possible.

Reply:
@jsuarezruiz  ,Since PageSheet functionality is available only on iOS, we have modified the test cases to ensure proper functionality.Please review and let me know if there are any additional concerns.